### PR TITLE
Rename all ltla/lad naming to "area"

### PR DESCRIPTION
**Pull request by theosanderson**
This genericises covince to be less fixed on the UK context. No functional changes. No need to actually read the code...

### CODE REVIEW

File: src/components/Chloropleth.jsx [hunk 1]
```diff
@@ -167,15 +168,16 @@ const Chloropleth = (props) => {
     }
 
     const features = tiles.features.map(f => {
-      const { lad19cd } = f.properties
```
Comment (richardgoater) [inline src/components/Chloropleth.jsx:170]: Like this change a lot, it was always way too specific. Will allow is to use the 2020 boundaries I guess?

File: src/components/Covid19.jsx [hunk 2]
```diff
@@ -54,13 +54,13 @@ const Covid19 = ({ lineColor = 'blueGray', tiles, data, dataPath }) => {
...
+    if (areaState.currentArea === 'overview') {
       return {
-        category: 'National overview',
-        heading: 'England',
+        category: data.overview.category,
+        heading: data.overview.heading,
         subheading: (
           <span className='flex items-center text-subheading'>
-            Explore local authorities {
+            Explore {data.overview.subnoun_plural} {
```
Comment (richardgoater) [inline src/components/Covid19.jsx:63]: getting into some ontology here! :)

File: src/components/Chloropleth.jsx (hunk 1 above)
Comment (theosanderson) [reply src/components/Chloropleth.jsx:170]: Yeah for the moment one has to tinker with the actual JSON, but in due course we could define the accessor instead

Comment (richardgoater) [reply src/components/Chloropleth.jsx:170]: ah right, interesting!

**Review by richardgoater — APPROVED** [review]
Nice one dude, like it a lot 👍🏽

**Pull request merged**